config/initializers/mailer.rb

```diff
@@ -5,8 +5,8 @@
     address:              'smtp.gmail.com',
     port:                 587,
     domain:               'socius.herokuapp.com',
-    user_name:            ENV["GMAIL_USERNAME"],
-    password:             ENV["GMAIL_PASSWORD"],
+    user_name:            'cmusocius@gmail.com',
```

Comment:
Don't expose account info online. By set the ENV variables, I meant in your environment, not in the code files. Change this back ASAP, and we'll have to remove any commit histories from the Github.